examples/cpp-debug-workspace/README.md

```diff
@@ -0,0 +1,24 @@
+## Eclipse Theia - `@theia/cpp-debug` example workspace
```

Comment:
I think this README would be easier to find it it were one or two levels up. Maybe integrate this in the README in the repo's root?

Reply:
I've added a reference to the example workspace at the `readme` located at the root of the repository.
I've described at a high level what it includes and provided a link to the actual `readme` of the example workspace as well. Is this sufficient?